.github/workflows/test.yml

```diff
@@ -26,6 +26,8 @@ on:
 env:
   ETCDCTL_API: "3"
   PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION: python
+  ETCD_VERSION: "3.5.0"
```

Comment:
It looks like they don't recommend 3.5.0 for production, so maybe it's worth testing against latest etcd version instead?

<img width="1074" alt="Screenshot 2023-02-22 at 10 58 54" src="https://user-images.githubusercontent.com/7579677/220571646-d7054d79-3d8f-49b4-a36e-24a5d044f864.png">

Reply:
3.5.0 is old, but 3.5.7 was out (and 3.4.24) --- they seem to maintain 3.4 and 3.5 actively.